portfolio/src/main/webapp/photos.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="navbar">
+        <ul>
+          <li><a href="index.html">home</a></li> 
+          <li><a href="projects.html">projects</a></li> 
+          <li><a href="blog.html">blog</a></li> 
+          <li><a href="photos.html">photos</a></li> 
+          <li><a href="random.html">random</a></li>
+        </ul>
+    </div> 
+    
+    <div id="photoscontent">
+        <div id="photosdescrip">
+            <h1>Photos + nice memories</h1>
+            <h4>Amateur photography shot on (arguably outdated) iPhone 6</h4> 
+        </div>
+        <div id="photosblocks">
+            <div class="photoblock" id="pr">
+                <img src="./images/gallery/pr.jpg">
```

Comment:
nit: for images, consider having an 'alt' tag -- mostly most beneficial for accessibility reasons but also for other reasons such as internet connectivity so there is alternate text instead of blank.